README.md

```diff
@@ -1,8 +1,7 @@
-# Codex.Editor server validation sample
+# Editor.js PHP
 
-This library allows you to use Codex.Editor server validation. 
-You can easily make your client plugins for Codex Editor and then 
-extend server Tool which will be able to clean dirty data or handle that.
+This library allows you to use EditorJS server validation. 
```

Comment:
Server-side implementation sample for the Editor.js. Contains validation and HTML sanitising logic and converts editor's JSON output to the useful Block objects.